apps/server/src/modules/lesson/service/lesson-copy.service.ts

```diff
@@ -227,20 +227,15 @@ export class LessonCopyService {
 				copiedContent.push(linkContent);
 				copiedContentStatus.push(embeddedTaskStatus);
 			}
-			if (element.component === ComponentType.NEXBOARD && nexboardEnabled) {
+			if (element.component === ComponentType.NEXBOARD) {
```

Comment:
Why did you remove `&& nexboardEnabled`? It'll now be executed always and we don't want that if the Nexboard is not enabled at all.